src/components/transcript/transcript.tsx

```diff
@@ -168,8 +169,20 @@ export class Transcript extends Component<TranscriptProps, TranscriptState> {
       this._resizeObserver?.disconnect();
       this._resizeObserver = null;
     }
+    document.removeEventListener('keydown', this._handleKeyboardShortcuts);
   }
 
+  private _handleKeyboardShortcuts = (event: KeyboardEvent) => {
+    if ((event.ctrlKey || event.metaKey) && event.code === 'KeyJ') {
+      event.preventDefault();
+      const jumpToSearchButton = document.querySelector('[data-testid="transcript_jumpToSearchMatch"]') as HTMLButtonElement;
```

Comment:
let's not use data-testid because:
1. If on page more than 1 player, you would get incorrect DOM node (from first player on page)
2. that's only for test purposes

Keep ref of button and focus it instead